### ISSUE TITLE
pip install pyrosm fails v0.6.1.

### ISSUE THREAD

**Issue by CaptainInler** [HTenkanen/pyrosm#211]
**Describe the bug**
```pip install pyrosm``` fails 😭

**To Reproduce**
I tried it in a docker container.  
1. ```docker run --rm -it python:3.11.0-slim-bullseye /bin/bash```
2. ```pip install pyrosm```

**Environment:**
 - OS: Debian GNU/Linux 11 (bullseye)
 - Python package source: PyPi
 - Versions of Python: 3.11.0

**Errormessage:**
```
root@24def12338e4:/# pip install pyrosm  
Collecting pyrosm
  Downloading pyrosm-0.6.1.tar.gz (2.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.1/2.1 MB 26.7 MB/s eta 0:00:00
  Installing build dependencies ... error
  error: subprocess-exited-with-error

  × pip subprocess to install build dependencies did not run successfully.
  │ exit code: 1
  ╰─> [338 lines of output]
      Collecting setuptools
        Downloading setuptools-66.1.1-py3-none-any.whl (1.3 MB)
           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.3/1.3 MB 32.7 MB/s eta 0:00:00
      Collecting wheel
        Downloading wheel-0.38.4-py3-none-any.whl (36 kB)
      Collecting Cython
        Downloading Cython-0.29.33-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.manylinux_2_24_x86_64.whl (1.9 MB)
           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.9/1.9 MB 79.0 MB/s eta 0:00:00
      Collecting cykhash
        Downloading cykhash-2.0.0.tar.gz (43 kB)
           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 43.3/43.3 kB 5.1 MB/s eta 0:00:00
        Installing build dependencies: started
        Installing build dependencies: finished with status 'done'
        Getting requirements to build wheel: started
        Getting requirements to build wheel: finished with status 'done'
        Preparing metadata (pyproject.toml): started
        Preparing metadata (pyproject.toml): finished with status 'done'
      Collecting pyrobuf
        Downloading pyrobuf-0.9.3.tar.gz (258 kB)
           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 258.5/258.5 kB 24.0 MB/s eta 0:00:00
        Preparing metadata (setup.py): started
        Preparing metadata (setup.py): finished with status 'done'
      Collecting jinja2>=2.8
        Using cached Jinja2-3.1.2-py3-none-any.whl (133 kB)
      Collecting MarkupSafe>=2.0
        Using cached MarkupSafe-2.1.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (27 kB)
      Building wheels for collected packages: cykhash, pyrobuf
        Building wheel for cykhash (pyproject.toml): started
        Building wheel for cykhash (pyproject.toml): finished with status 'error'
        error: subprocess-exited-with-error

        × Building wheel for cykhash (pyproject.toml) did not run successfully.
        │ exit code: 1
        ╰─> [131 lines of output]
            running bdist_wheel
            running build
            running build_py
            creating build
            creating build/lib.linux-x86_64-cpython-311
            creating build/lib.linux-x86_64-cpython-311/cykhash
            copying src/cykhash/__init__.py -> build/lib.linux-x86_64-cpython-311/cykhash
            copying src/cykhash/compat.py -> build/lib.linux-x86_64-cpython-311/cykhash
            running egg_info
            writing src/cykhash.egg-info/PKG-INFO
            writing dependency_links to src/cykhash.egg-info/dependency_links.txt
            writing top-level names to src/cykhash.egg-info/top_level.txt
            reading manifest file 'src/cykhash.egg-info/SOURCES.txt'
            reading manifest template 'MANIFEST.in'
            adding license file 'LICENSE'
            writing manifest file 'src/cykhash.egg-info/SOURCES.txt'
            /tmp/pip-build-env-6lhm0ohd/overlay/lib/python3.11/site-packages/setuptools/command/build_py.py:202: SetuptoolsDeprecationWarning:     Installing 'cykhash.maps' as data is deprecated, please list it in `packages`.
                !!

                ############################
                # Package would be ignored #
                ############################
                Python recognizes 'cykhash.maps' as an importable package,
                but it is not listed in the `packages` configuration of setuptools.

                'cykhash.maps' has been automatically added to the distribution only
                because it may contain data files, but this behavior is likely to change
                in future versions of setuptools (and therefore is considered deprecated).

                Please make sure that 'cykhash.maps' is included as a package by using
                the `packages` configuration field or the proper discovery methods
                (for example by using `find_namespace_packages(...)`/`find_namespace:`
                instead of `find_packages(...)`/`find:`).

                You can read more about "package discovery" and "data files" on setuptools
                documentation page.


            !!

......

```

**Comment by leonefamily** [comment]
I was having the same issue as well (it complains about not being able to find `Python.h`), and managed to resolve it by installing package python3-dev as root user on Pop!_OS (which is Debian/Ubuntu based). Here are different ways to install it on different Linux distros (taken from https://stackoverflow.com/a/21530768):

For `apt` (Ubuntu, Debian...):
`sudo apt-get install python3-dev`

For `yum` (CentOS, RHEL...):

`sudo yum install python3-devel`

For `dnf` (Fedora...):

`sudo dnf install python3-devel`

For `zypper` (openSUSE...):

`sudo zypper in python3-devel`

For `apk` (Alpine...):

`sudo apk add python3-dev` 

For `apt-cyg` (Cygwin...):

`apt-cyg install python3-devel`

**Comment by HTenkanen** [comment]
Installation issues are now fixed in the new release pyrosm v0.6.2. Try updating `pyrosm` with pip (installing from conda-forge is soon available as well).
